setup.cfg

```diff
@@ -1,33 +1,85 @@
-[build_sphinx]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
-
-[upload_docs]
-upload-dir = docs/_build/html
-show-response = 1
-
-[tool:pytest]
-addopts = --doctest-ignore-import-errors
-minversion = 3.0
-norecursedirs = build docs/_build
-doctest_plus = enabled
-
-[ah_bootstrap]
-auto_use = True
-
 [metadata]
-package_name = astroplan
-description = Observation planning package for astronomers
-long_description = Observation planning package for astronomers
+name = astroplan
 author = Astroplan developers
 author_email = astropy-dev@googlegroups.com
 license = BSD
+license_file = LICENSE.rst
 url = https://github.com/astropy/astroplan
+description = Observation planning package for astronomers
+long_description = file: README.rst
+long_description_content_type = text/x-rst
 edit_on_github = False
 github_project = astropy/astroplan
 
-[entry_points]
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.6
```

Comment:
@bmorris3 @wtgee -- I took a guess here as to the correct version. I would recommend not supporting Python 2 anymore, and even python 3.5 is pretty old at this point.

Reply:
i would set minimum to 3.7 at this point unless there's a specific reason to support 3.6. iirc, astropy has dropping support for 3.6 in its timeline, at least.